python/lsst/rapid/analysis/utils.py

```diff
@@ -177,7 +184,7 @@ def humanNameForCelestialObject(objName):
 def _getAltAzZenithsFromSeqNum(butler, dayObs, seqNumList):
     azimuths, elevations, zeniths = [], [], []
     for seqNum in seqNumList:
-        md = butler.get("raw_md", dayObs=dayObs, seqNum=seqNum)
+        md = butler.get('raw.metadata', day_obs=dayObs, seq_num=seqNum, detector=0)
         elevations.append(md['ELSTART'])
```

Comment:
Please don't use direct header access for things that the metadata translator understands.

Reply:
Yup, very old code. All gone.